src/visualization/include/iDynTree/Visualizer.h

```diff
@@ -333,10 +334,16 @@ struct VisualizerOptions
      * Initial height (in pixels) of the created window (default: 600).
      */
     int winHeight;
+    
+    /**
+     * Dimension of the root frame arrows
+     */
+    double arrowsDimension;
```

Comment:
Can you add "root frame" to the param name? Something like rootFrameArrowsDimension I think would be ok. Please mention in the docs that the parameters is expressed in meters.